### PR TITLE
feat: experimentation-ess  audit

### PR DESCRIPTION
**Pull request by rpapani**
This PR adds two audits for experimentation-ess
- experimentation-ess-monthly - audit of all the experiments running in the last 30 days
- experimentation-ess-daily - audit of all the experiments running in the last 1 day - used in graph data for experiments in ESS

## Experiments schema in the audit

```
[{
  "id": "string",
  "name": "string",
  "type": "string",
  "url": "string",
  "status": "string",
  "startDate": "string",
  "endDate": "string",
  "conversionEventName": "string - comes from page",
  "conversionEventValue": "string - comes from page",
  "variants": [
    {
      "name": "string",
      "split": "number",
      "url": "string",
      "p_value": "number",
      "views": "number",
      "metrics": [
        {
          "type": "click",
          "selector": "string",
          "value": "number"
        },
        {
          "type": "conversion",
          "selector": "string",
          "value": "number"
        },
        {
          "type": "formsubmit",
          "selector": "string",
          "value": "number"
        }
      ]
    }
  ]
}]
```

## Related PRs
https://github.com/adobe/spacecat-shared/pull/264

### Note
- tests skipped for the current scope

### CODE REVIEW

**Review by ramboz — APPROVED** [review]

File: src/ess-experimentation/handler.js [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+import { JSDOM } from 'jsdom';
+import { AuditBuilder } from '../common/audit-builder.js';
+
+const EXPERIMENT_PLUGIN_OPTIONS = {
+  experimentsRoot: '/experiments',
+  experimentsConfigFile: 'manifest.json',
+  experimentsMetaTag: 'experiment',
+  experimentsQueryParameter: 'experiment',
+};
+const SPACECAT_RUM_API_ENDPOINT = 'https://spacecat.experiencecloud.live/api/v1/rum';
```
Comment (ekremney) [inline src/ess-experimentation/handler.js:24]: we decided not to expose the rum stuff yet due to some constraints. This endpoint doesn't exist currently. For now, we can just set the `fullAuditRef` to domain.

File: src/ess-experimentation/handler.js [hunk 2]
```diff
@@ -0,0 +1,477 @@
...
+        } else {
+          variants[camelVN][key] = line[vn];
+        }
+      });
+    });
+    config.variants = variants;
+    config.variantNames = variantNames;
+    return config;
+  } catch (e) {
+    // eslint-disable-next-line no-console
```
Comment (ekremney) [inline src/ess-experimentation/handler.js:153]: I guess this eslint rule and other `no-console` ones are not needed

**Review by ekremney — APPROVED** [review]
This is by far the most complex audit we had 🥇 

**note for the post-july**: we should definitely add test cases to make sure that the logic here works and guarded for future iterations

**Pull request merged**